crates/re_viewer/Cargo.toml

```diff
@@ -87,6 +87,7 @@ egui.workspace = true
 egui-wgpu.workspace = true
 image = { workspace = true, default-features = false, features = ["png"] }
 itertools = { workspace = true }
+lazy_static = { workspace = true }
```

Comment:
We usually prefer [`once_cell`](https://docs.rs/once_cell/latest/once_cell/)

If we update MSRV we can probably get rid of that too: https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html#oncecell-and-oncelock